pom.xml

```diff
@@ -41,252 +41,20 @@
-        <icanhandlebarz.version>0.1</icanhandlebarz.version>
-        <iframe-resizer.version>2.6.2</iframe-resizer.version>
-        <jknack.handlebars.version>2.0.0</jknack.handlebars.version>
-        <jquery.version>1.11.0</jquery.version>
-        <jquery-ui.version>1.10.4</jquery-ui.version>
-        <jruby.version>9.0.4.0</jruby.version>
-        <marionette.version>2.4.5</marionette.version>
-        <picocontainer.version>1.3_1</picocontainer.version>
-        <requirejs-plugins.version>1.0.2</requirejs-plugins.version>
-        <underscore.version>1.8.3</underscore.version>
-        <yarn.version>v1.5.1</yarn.version>
-
-        <!--Test properties-->
-        <hamcrest-all.servicemix.version>1.3_1</hamcrest-all.servicemix.version>
-        <hamcrest-all.version>1.3</hamcrest-all.version>
-        <hamcrest-junit.version>2.0.0.0</hamcrest-junit.version>
-        <hamcrest-optional.version>1.0</hamcrest-optional.version>
-        <junit.version>4.12</junit.version>
-        <mockito-core.version>2.8.47</mockito-core.version>
-        <mockito-all.version>1.10.19</mockito-all.version>
-        <powermock.version>1.6.4</powermock.version>
-
-        <!--Maven properties-->
         <dependency-check-maven.version>3.1.1</dependency-check-maven.version>
         <jbake.maven.plugin.version>0.0.9</jbake.maven.plugin.version>
-        <karaf.plugin.version>4.2.1</karaf.plugin.version>
+        <maven-checkstyle-plugin.version>3.0.0</maven-checkstyle-plugin.version>
         <maven.clean.plugin.version>2.4.1</maven.clean.plugin.version>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven-dependency-plugin.version>3.0.2</maven-dependency-plugin.version>
         <maven.deploy.plugin.version>2.6</maven.deploy.plugin.version>
-        <maven.install.plugin.version>2.3.1</maven.install.plugin.version>
+        <maven.install.plugin.version>2.5.2</maven.install.plugin.version>
+        <maven-jacoco-plugin.version>0.8.1</maven-jacoco-plugin.version>
         <maven.javadoc.plugin.version>2.7</maven.javadoc.plugin.version>
+        <maven-jaxb2-plugin.version>0.8.2</maven-jaxb2-plugin.version>
         <maven.release.plugin.version>2.5.1</maven.release.plugin.version>
         <maven.site.plugin.version>3.4</maven.site.plugin.version>
-        <maven-checkstyle-plugin.version>3.0.0</maven-checkstyle-plugin.version>
-        <maven-dependency-plugin.version>3.0.2</maven-dependency-plugin.version>
-        <maven-install-plugin.version>2.5.2</maven-install-plugin.version>
-        <maven-jacoco-plugin.version>0.8.1</maven-jacoco-plugin.version>
-        <maven-jaxb2-plugin.version>0.8.2</maven-jaxb2-plugin.version>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
         <maven-war-plugin.version>2.2</maven-war-plugin.version>
-
-        <!--Project properties-->
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
✏️ I think these four properties are probably safe to keep here. At least two encoding properties.

Reply:
good call